lib/features/composer/domain/usecases/save_composer_cache_on_web_interactor.dart

```diff
@@ -15,10 +17,18 @@ class SaveComposerCacheOnWebInteractor {
     this._composerRepository,
   );
 
-  Future<Either<Failure, Success>> execute(CreateEmailRequest createEmailRequest) async {
+  Future<Either<Failure, Success>> execute(
+    CreateEmailRequest createEmailRequest,
+    AccountId accountId,
+    UserName userName  
+  ) async {
     try {
       final emailCreated = await _composerRepository.generateEmail(createEmailRequest);
-      _composerCacheRepository.saveComposerCacheOnWeb(emailCreated);
+      await _composerCacheRepository.saveComposerCacheOnWeb(
+        emailCreated,
+        accountId: accountId,
+        userName: userName,
+        identity: createEmailRequest.identity);
```

Comment:
Also replace `CreateEmailRequest` to `Identity ` in parameter

Reply:
Done